output/client.go

```diff
@@ -14,15 +14,15 @@ import (
 
 const (
 	defaultURL                = "https://listener.logz.io:8071"
+	defaultId                 = "1"
```

Comment:
Can you change the default value to be something like logzio_output1. 1 is to generic and maybe there is this kind of id in the system already